src/scripts/main.js

```diff
@@ -1,3 +1,164 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+// const table = document.querySelector('table');
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tr = tbody.getElementsByTagName('tr');
+const th = Array.from(document.getElementsByTagName(`th`));
+let indexSort;
+
+// add form
+
+const form = document.createElement('form');
+const button = document.createElement('button');
+
+button.textContent = 'Save to table';
+button.type = 'submit';
+form.classList.add('new-employee-form');
+body.append(form);
+
+form.insertAdjacentHTML(
+  'beforeend',
+  '<label>Name: <input data-qa="name" name="name" type="text"></label>',
+);
+
+form.insertAdjacentHTML(
+  'beforeend',
+  `<label>
+  Position:
+  <input data-qa="position" name="position" type="text">
+</label>`,
+);
+
+form.insertAdjacentHTML(
+  'beforeend',
+  `<label>
+  Office:
+  <select data-qa="office" name="office" type="text"></select>
+  </label>`,
+);
+
+form.insertAdjacentHTML(
+  'beforeend',
+  '<label>Age: <input data-qa="age" name="age" type="number"></label>',
+);
+
+form.insertAdjacentHTML(
+  'beforeend',
+  '<label>Salary: <input data-qa="salary" name="salary" type="number"></label>',
+);
+
+const locations = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+locations.forEach((locat) => {
+  const option = document.createElement('option');
+
+  option.setAttribute('locat', locat);
```

Comment:
The attribute 'locat' is being set on the option element, which is incorrect. You should use 'value' instead to set the option's value attribute.